examples/ukernels/cpu_brgemm.cpp

```diff
@@ -66,8 +66,17 @@ void brgemm_example() {
     // packing requirements once for multiple objects.
     // Based on this information, specific `ldb` value can be used, since
     // transform has a limited set of values supported.
-    const bool need_pack
-            = brgemm::get_B_pack_type(a_dt, b_dt) == pack_type::pack32;
+    bool need_pack = false;
+    try {
+        need_pack = brgemm::get_B_pack_type(a_dt, b_dt) == pack_type::pack32;
+    } catch (error &e) {
+        if (e.status == dnnl_unimplemented)
+            throw example_allows_unimplemented {
+                    "Kernel is not supported on this platform.\n"};
+
+        // on any other error just re-throw
+        throw;
```

Comment:
```suggestion
        std::rethrow_exception(std::current_exception());
```

Reply:
I'll keep this as is since they are going to be removed.